### ISSUE TITLE
Differences in parameter correlation matrix

### ISSUE THREAD

**Issue by paulray** [nanograv/PINT#684]
Below are three correlation matrix outputs from TEMPO, Tempo2 and PINT.
A couple things to note:
* The sign of the correlations for some, but not all, parameters are different in PINT than they are in TEMPO or Tempo2 (which agree). It is a bit agonizing to compare since the order of params is different, but it seems like the correlations with the position parameters are the ones with opposite signs.
* Tempo2 output include an explicit offset parameter, which is something we've been considering in PINT

TEMPO:
```
                f0    f1   Dec    RA    DM
            --------------------------------
  1.     f0|   1.00
  2.     f1|  -0.80  1.00
  3.    Dec|  -0.25  0.34  1.00
  4.     RA|   0.09 -0.29 -0.07  1.00
  5.     DM|  -0.01 -0.06  0.06 -0.00  1.00

       gcor:,  0.82  0.84  0.36  0.39  0.15
            --------------------------------
                f0    f1   Dec    RA    DM
            --------------------------------
```
Tempo2:
```
Param      	YOFF       	RAJ        	DECJ       	F0         	F1         	DM         	
YOFF       	+1.00000000	
RAJ        	+0.02479577	+1.00000000	
DECJ       	-0.05720205	-0.07221980	+1.00000000	
F0         	+0.00911126	+0.08729879	-0.24713622	+1.00000000	
F1         	+0.01940102	-0.29426662	+0.34386335	-0.79767127	+1.00000000	
DM         	-0.99531307	-0.00496850	+0.06481275	-0.00674241	-0.05755869	+1.00000000	
```
PINT
```
Parameter correlation matrix:
         RAJ   DECJ    DM     F0     F1   
  RAJ   1.000 -0.072 -0.005 -0.087  0.294 
 DECJ  -0.072  1.000  0.065  0.247 -0.344 
  DM   -0.005  0.065  1.000  0.007  0.058 
  F0   -0.087  0.247  0.007  1.000 -0.798 
  F1    0.294 -0.344  0.058 -0.798  1.000 
```

**Comment by scottransom** [comment]
I noticed the PINT vs TEMPO sign differences when I made the pretty printer.  I'd be willing to bet that ours are correct (given our deriv calculations and the fact that I don't think that anyone did anything with TEMPO's but look at them), but it would be good to verify that.  I'm not sure how, though...

What is the offset parameter?  Is that something to do with the absolute phase?  If so, we do have that as an option (at least in my pretty printer).

**Comment by luojing1211** [comment]
Here is why the derivatives have a negative sign. It also has an explanation in the comment. 

https://github.com/nanograv/PINT/blob/f9ee990aa09f94d566816929d6b071e5e7a5c5c2/src/pint/models/timing_model.py#L1062

**Comment by paulray** [comment]
@scottransom Yes I think the offset is just the hidden absolute phase parameter in the design matrix.

@luojing1211 I see how that could explain an opposite sign convention, but PINT has *some* values opposite sign and others not.
